ExerciseApp/ExerciseApp/SplashActivity.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using ExerciseApp.Data;
+
+namespace ExerciseApp
+{
+    [Activity(Label = "ExerciseApp", Icon = "@drawable/icon", Theme = "@style/SplashScreen", MainLauncher = true, NoHistory = true)]
+    public class SplashActivity : Activity
+    {
+        private readonly Database _db = new Database("exercise.db3");
```

Comment:
Inject Database interface into class to make it unit testable

e.g:

public SplashActivity(IDatabase db)
{
   this._db = db
}